dnsapi/README.md

```diff
@@ -1172,6 +1172,32 @@ acme.sh --issue --dns dns_doapi -d example.com -d *.example.com
 
 The API token will be saved in `~/.acme.sh/account.conf` and will be reused when needed.
 
+## 61. Use MyDevil.net
+
+Make sure that you can execute own binaries:
+
+```sh
+devil binexec on
+```
+
+Install acme.sh somewhere on your mydevil host account, probably in your home directory.
+Once it is installed, add it to your `~/bin` directory (and make sure it exists first):
+
+```sh
+mkdir ~/bin
+ln -s /path/to/installed/directory/of/.acme.sh/acme.sh ~/bin/acme.sh
```

Comment:
It's net necessary to link in the bin folder.

Reply:
It's not necessary, but it makes all the commands in examples a lot easier to read :).
Also `~/bin` is supported by default by MyDevil.net, e.g., `PATH` includes it by default.

Reply:
@Neilpang i rewrote that part of README.